### ISSUE TITLE
Configuring deno.enable = false always show the error.

### ISSUE THREAD

**Issue by LumaKernel** [fannheyward/coc-deno#184]
Hi 💫 

```ts
import { workspace } from 'coc.nvim';

// ...
  workspace.getConfiguration('deno').update('enable', false);
// ...
```

Firing above anyway after running coc-deno, it shows always the error. I checked this by registering as command including just this line like following, and just fire it with `CocCommand myext.test`.

```ts
// ...
  commands.registerCommand(`${EXTENSION_NS}.test`, () => {
    workspace.getConfiguration('deno').update('enable', false);
  });
// ...
```

## Error

<img width="426" alt="image" src="https://user-images.githubusercontent.com/29811106/194862683-e13628f5-2ca1-4c6c-a1ca-816f2531c516.png">

```
[coc.nvim]: UnhandledRejection: Starting server failed
Error: Starting server failed
    at LanguageClient.$start (/home/luma/.cache/dein/nvim-huge--coc--coc--lsp/repos/github.com/neoclide/coc.nvim_master/build/index.js:54056:17)
    at processTicksAndRejections (internal/process/task_queues.js:95:5)
    at async LanguageClient.sendNotification (/home/luma/.cache/dein/nvim-huge--coc--coc--lsp/repos/github.com/neoclide/coc.nvim_master/build/index.js:53769:30)
Press ENTER or type command to continue
```

## Workaround

I confirmed it cannot occur after commenting out restart on change configuration.

https://github.com/fannheyward/coc-deno/compare/master...LumaKernel:coc-deno-pruned:master?expand=1

## Suggestions

- Restart only after enabling.
- Suppress restarting by configuration.
  - I may prefer this one. maybe both would work, but I want to control restarting timing by my side.

## Versions

- `coc.nvim` : recent (Oct 1) master `354c751a6d767ff62cb2774df7d50d8ab1fb81e1`
- `coc-deno` : latest from npm (3.10.1)

---

Thanks in advance.

**Comment by LumaKernel** [comment]
And I couldn't workaround by `CocDisable` -> (configure time) -> `CocEnable`.

**Comment by fannheyward** [comment]
Reproduced.

**Comment by fannheyward** [comment]
https://github.com/denoland/vscode_deno/blob/8581e36920cc5c21cd3e43e2e94396b67b412b19/client/src/extension.ts#L141-L145

**Comment by fannheyward** [comment]
Suppress the restarting on configurations changed.

**Comment by fannheyward** [comment]
> Suppress the restarting on configurations changed.

Released in v3.10.2
